python/python/res/simulator/batch_simulator.py

```diff
@@ -95,6 +102,7 @@ def start(self, case_name, controls):
         batch complete before you start a new batch.
         """
         ens_config = self.res_config.ensemble_config
+        self.ert.addDataKW("<CASE_NAME>", _slug(case_name))
```

Comment:
I would like that we make `<CASE_NAME>` a part of the API of the `BatchSimulator` by documenting the behavior.